cel/cel_test.go

```diff
@@ -305,3 +305,45 @@ func Test_EvalOptions(t *testing.T) {
 		t.Errorf("Got '%v', expected 'true'", rhsVal)
 	}
 }
+
+func Benchmark_EvalOptions(b *testing.B) {
+	e, _ := NewEnv(
+		Declarations(
+			decls.NewIdent("ai", decls.Int, nil),
+			decls.NewIdent("ar", decls.NewMapType(decls.String, decls.String), nil),
+		),
+	)
+	past, _ := e.Parse("ai == 20 || ar['foo'] == 'bar'")
+	cast, _ := e.Check(past)
+	vars := Vars(map[string]interface{}{
+		"ai": 2,
+		"ar": map[string]string{
+			"foo": "bar",
+		},
+	})
+
+	b.Run("track-state", func(bb *testing.B) {
+		prg, _ := e.Program(cast, EvalOptions(OptTrackState))
```

Comment:
Hi! This could be smaller if you loop over the EvalOptions parameter (or create a closure with it)

Reply:
Done.